File: python/rna-transcription/rna_transcription.py
dna_rna = {
    "A": "U",
    "G": "C",
    "C": "G",
    "T": "A",
}

def to_rna(dna_strand):
    return "".join(list(map(lambda x: dna_rna.get(x, ""), dna_strand)))

File: javascript/rna-transcription/rna-transcription.js
const DNA_RNA = {
  C: "G",
  G: "C",
  T: "A",
  A: "U",
};

export const toRna = (dna) =>
  dna
    .toUpperCase()
    .split("")
    .map((char) => DNA_RNA[char] || char)
    .join("");
